prime-router/metadata/hl7_mapping/ORU_R01/base/observation-result.yml

```diff
@@ -155,29 +155,29 @@ elements:
     hl7Spec: [ '%{hl7OBXField}-19' ]
 
   - name: performing-organization-name-pracrole
-    condition: '%observation.performer.resolve() is PractitionerRole'
-    resource: '%resource.performer.resolve().organization.resolve()'
+    condition: '%observation.performer.resolve().ofType(PractitionerRole).exists()'
```

Comment:
What determined whether you used `x.y.exists and x.y is Z` vs `x.y.ofType(Z).exists`? They seem similar, maybe the latter makes more sense for repeating components? Wasn't sure so wanted to check if you had more explanation.

Reply:
So this one was an interesting and maybe deserves a second look.  For some of the integration tests, `performer` was an array with two elements, one PractictionerRole and one Organization.  This check was to make sure that the condition only evaluates to true if we can get a PractitionerRole

Reply:
Ah okay, guessing this used to work because the resource would only be valid if the condition was true anyway. As long as integration tests work, this seems fine, though yea I am curious about a few things here, might add another comment or two.